docs/4-Transition/lessons.md

```diff
@@ -143,4 +158,87 @@ When `nginx.ingress.kubernetes.io/proxy-body-size` is set to 0, this removes the
+
+### Logging Into Navigator Fails and Displays A Network Error Message
+
+#### The Problem
+
+While the team was working on use cases with the client, we ran into an issue when trying to log into Navigator. When attempting to log into Navigator, the team was receiving the following error message:
+
+![FileNet Lessons Learn 3.png](https://zenhub.ibm.com/images/6442f46ac0371b5acaba3fc4/6cd0cd1b-291c-4983-85b0-12083f4b6ff0)
+
+This message appears to be the general error message for networking issues within FileNet. The team then decided to inspect the Navigator pod logs. First we need to bash into the pod. 
+
+```
+kubectl exec -it NAVIGATOR_POD_NAME -- bash
+```
+
+Once inside the Navigator pod, there should be a directory name `logs`. Change to that directory. Inside the logs directory, there might be multiple folders. Look for the folder with the same name as the currently running pod. That folder should contain the latest logs. When the team examined those logs we saw the following error message:
+
+```
+com.filenet.api.exception.EngineRuntimeException: FNRCA0031E: API_UNABLE_TO_USE_CONNECTION: The URI for server communication cannot be determined from the connection object URL
+```
+
+Additionally, the team was given another clue when we were able to log in to the Navigator admin desktop. You can access the admin desktop by appending `?desktop=admin` to the end of the Navigator URL. Logging into the default desktop, however, continued to throw a network error. 
+
+The team then checked the Server URL associated with the Repositories. To do this, log into the Navigator admin desktop. Click "Repositories" from the menu on the left side. Next select the repo you want to verify the connection to. Once selected it should highlight. Now press the "Edit" button directly above. 
+
+![FileNet Lessons Learn 10.png](https://zenhub.ibm.com/images/6442f46ac0371b5acaba3fc4/5474ab10-9627-4a79-ad5b-056fbb8cfa26)
+
+The next page will display the general settings, along with the Server URL for that repository. Here you can edit and make changes to the repository settings. If all the options are correct, click the "Connect..." button at the bottom.
+
+![FileNet Lessons Learn 10.png](https://zenhub.ibm.com/images/6442f46ac0371b5acaba3fc4/c47f3844-6720-472b-b862-3873df2e44ac)
+
+In our case, when we hit the connect button, we saw the same network error message we saw at the beginning. 
+
+Alternatively, We can also view some of the Repository settings in the CR.
+
+```yaml
+ic_icn_init_info:
+  icn_repos:
+    - add repo id: "OS01геро"
+      // highlight-next-line
+      add_repo_ce_wsi_url: "http://{{ meta.name }}-cpe-stateless-svc.{{ meta.namespace }}.svc:9080/wsi/FNCEWS40MTOM/"
```

Comment:
This was a change from 5.5.11 and up. I referenced this elsewhere in the docs, so let's reiterate what I put there and include a link to it here as something we've added.

Reply:
Updated awaiting further feedback.